src/extension/popup.js

```diff
@@ -333,7 +356,8 @@ function populateCharacter(response) {
             // When loading settings, the discord target combobox gets replaced in order to be filled,
             // so we need to fetch it again to add the advanced-option class to it
             $("#beyond20-option-discord-target").addClass("advanced-option");
-        }
+        
+        options.find(".effects-option").hide();}
```

Comment:
Late night coding and I just noticed this in code, the line seems to have been written before the closing `}` instead of after... so it's inside of the `if` scope above it.
Fixed it, pushing soon!